src/components/atoms/CreatableMultiSelect/CreatableMultiSelect.tsx

```diff
@@ -62,6 +65,10 @@ const CreatableMultiSelect: React.FC<MultiSelectProps> = props => {
     }
   };
 
+  useDebounce(() => {
+    setApplyValidationStyles(validation);
+  }, 100, [validation]);
```

Comment:
Why it should be delayed? Shouldn't it use `validation` as before?

Reply:
Default antd fields have this delay. When labels and any input field is on the screen, validation is triggered different